api.py

```diff
@@ -82,7 +89,18 @@ def fetch_task_gql(mentor_id: str, question_id) -> GQLQueryBody:
     return {
         "query": """query UploadTask($mentorId: ID!, $questionId: ID!) {
             uploadTask(mentorId: $mentorId, questionId: $questionId){
-                taskList {
+                transcodeWebTask {
+                    task_name
```

Comment:
you probably dont need this anymore, it was used with the list to identify task types

Reply:
make sense

Reply:
[846abb0](https://github.com/mentorpal/mentor-upload-processor/pull/2/commits/846abb023e59f62ebf58924c9a089a13b536f0ae)